### ISSUE TITLE
Can't install packages all of a sudden

### ISSUE THREAD

**Issue by mzoz** [Homebrew/homebrew-core#83459]
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```shell
▶ brew config
HOMEBREW_VERSION: 3.2.8
ORIGIN: https://github.com/Homebrew/brew
HEAD: f026dd21c195952747395ebdda24327204f012ec
Last commit: 4 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: b328366cf329790563f8835ed148256e2e151860
Core tap last commit: 72 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 12
http_proxy: http://localhost:3128
https_proxy: http://localhost:3128
no_proxy: localhost
Homebrew Ruby: 2.6.3 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: dodeca-core 64-bit kabylake
Clang: 12.0.5 build 1205
Git: 2.30.1 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.64.1 => /usr/bin/curl
macOS: 11.5.1-x86_64
CLT: 12.5.0.0.1.1617976050
Xcode: 12.5 => /Applications/Xcode-12.5.app/Contents/Developer

~                                                                                                                    
▶ brew doctor
Your system is ready to brew.
```

### 

- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.

### What were you trying to do (and why)?

Install `qemu` package

### What happened (include all command output)?

```
Installing qemu
==> Downloading https://ghcr.io/v2/homebrew/core/gettext/manifests/0.21
Error: undefined method `first' for "Accept: application/vnd.oci.image.index.v1+json":String
Please report this issue:
  https://docs.brew.sh/Troubleshooting
/usr/local/Homebrew/Library/Homebrew/download_strategy.rb:394:in `block in fetch'
/usr/local/Homebrew/Library/Homebrew/download_strategy.rb:394:in `delete_if'
/usr/local/Homebrew/Library/Homebrew/download_strategy.rb:394:in `fetch'
/usr/local/Homebrew/Library/Homebrew/resource.rb:142:in `fetch'
/usr/local/Homebrew/Library/Homebrew/software_spec.rb:358:in `fetch_tab'
/usr/local/Homebrew/Library/Homebrew/formula.rb:2339:in `fetch_bottle_tab'
/usr/local/Homebrew/Library/Homebrew/formula_installer.rb:1128:in `fetch'
/usr/local/Homebrew/Library/Homebrew/formula_installer.rb:664:in `fetch_dependency'
/usr/local/Homebrew/Library/Homebrew/formula_installer.rb:1118:in `block in fetch_dependencies'
/usr/local/Homebrew/Library/Homebrew/formula_installer.rb:1118:in `each'
/usr/local/Homebrew/Library/Homebrew/formula_installer.rb:1118:in `fetch_dependencies'
/usr/local/Homebrew/Library/Homebrew/formula_installer.rb:1123:in `fetch'
/usr/local/Homebrew/Library/Homebrew/install.rb:299:in `install_formula'
/usr/local/Homebrew/Library/Homebrew/cmd/install.rb:207:in `block in install'
/usr/local/Homebrew/Library/Homebrew/cmd/install.rb:205:in `each'
/usr/local/Homebrew/Library/Homebrew/cmd/install.rb:205:in `install'
/usr/local/Homebrew/Library/Homebrew/brew.rb:129:in `<main>'
Installing qemu has failed!
```

### What did you expect to happen?

I uninstalled and reinstalled `brew` but still got the same problem, can't install other packages too.

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
brew install <anything>
```

**Comment by mzoz** [comment]
Worked yesterday and today when I tried to install `nginx` the same error message pops up.

**Comment by Bo98** [comment]
Something is causing the first url to redirect (it normally doesn't), which is why the issue seems to be unique to you. I have fixed the error for the next release (3.2.9) however.

**Comment by mzoz** [comment]
Thanks @Bo98 closing it now.
